deploy/sre-prometheus/100-telemeter-client.PrometheusRule.yaml

```diff
@@ -11,8 +11,10 @@ spec:
   - name: sre-telemeter-client
     rules:
     - alert: MetricsClientSendFailingSRE
-      expr: absent(rate(metricsclient_request_send{status_code="200"}[5m]) > 0) or rate(metricsclient_request_send{status_code!="200"}[5m]) > 0
-      for: 15m
+      expr: 100 * sum by (job) (rate(metricsclient_request_send{status_code="5xx"}[5m])) / sum by (job) (rate(metricsclient_request_send[5m])) > 14.4 * 10 and 100 * sum by (job) (rate(metricsclient_request_send{status_code="5xx"}[1h])) / sum by (job) (rate(metricsclient_request_send[1h])) > 14.4 * 10
```

Comment:
Can we still get a brief, human readable explanation in a yaml comment (like https://github.com/openshift/managed-cluster-config/blob/20d225db7bb8dcc79e12dc9a2490b84478c844d3/deploy/sre-prometheus/100-managed-velero-operator.PrometheusRule.yaml#L18-L20), and then suggesting that for a more detailed explanation, see the readme?

Reply:
We could. I'm not sure how to condense that so that it's meaningful but short enough for a comment.

Reply:
any idea/recommendation @cblecker ?